cpp/tests/join/join_tests.cpp

```diff
@@ -525,6 +525,96 @@ TEST_F(JoinTest, InnerJoinNoNulls)
   cudf::test::expect_tables_equal(*sorted_gold, *sorted_result);
 }
 
+TEST_F(JoinTest, InnerJoinNonAlignedCommon)
+{
+  column_wrapper<int32_t> col0_0{{3, 1, 2, 0, 2}};
```

Comment:
Just to reduce number of lines, nothing else
```suggestion
  cols0.push_back(column_wrapper<int32_t> ({3, 1, 2, 0, 2}).release());
```

Reply:
Addressed.